kalite/distributed/settings.py

```diff
@@ -160,17 +159,13 @@ def USER_FACING_PORT():
 # Cache is activated in every case,
 #   EXCEPT: if CACHE_TIME=0
 if CACHE_TIME != 0:  # None can mean infinite caching to some functions
-    # We can bump the version and forget about version info
-    version_info = version.VERSION_INFO()
-    if version.VERSION in version_info:
-        KEY_PREFIX = version_info[version.VERSION]["git_commit"][0:6]  # new cache for every build
-    else:
-        KEY_PREFIX = "unknown_version"
+    # When we change versions, cache changes, too
+    KEY_PREFIX = ".".join(version.VERSION)
```

Comment:
For anyone wondering like me, here's what [KEY_PREFIX](https://docs.djangoproject.com/en/1.5/ref/settings/#key-prefix) does. [See also](https://docs.djangoproject.com/en/1.5/topics/cache/#cache-key-prefixing).